test/functional/basic/gesture-e2e-specs.js

```diff
@@ -1,310 +1,363 @@
+        });
+        it('should tap on arbitrary coordinates with action', async function () {
+          const el = await driver.$(`~${BTN_OK_CNCL}`);
+          const loc = await el.getLocation();
+          const size = await el.getSize();
+
+          await driver.action('pointer')
+            .move(loc.x + size.width / 2, loc.y + size.height / 2)
+            .down()
+            .pause(100)
+            .up()
+            .perform();
+
+          await exitModal('OK');
+        });
+      });
+
+      describe('using touchAction', function () {
+        it('should tap on the element with touchAction', async function () {
+          const el = await driver.$(`~${BTN_OK_CNCL}`);
+          await driver.touchAction({
+            action: 'tap',
+            element: el,
+          });
+          await exitModal('OK');
+        });
+        it('should tap on arbitrary coordinates with action', async function () {
+          const el = await driver.$(`~${BTN_OK_CNCL}`);
+          const loc = await el.getLocation();
+          const size = await el.getSize();
+
+          await driver.touchAction({
+            action: 'tap',
+            x: loc.x + size.width / 2,
+            y: loc.y + size.height / 2,
+          });
+
+          await exitModal('OK');
+        });
+
+        it('should long press on an element', async function () {
+          const el = await driver.$(`~${BTN_OK_CNCL}`);
+          await driver.touchAction([
+            { action: 'longPress', element: el },
+            'release',
+          ]);
+
+          await exitModal('Cancel');
+        });
+        it('should long press on an element with duration through press-wait-release', async function () {
+          const el = await driver.$(`~${BTN_OK_CNCL}`);
+          await driver.touchAction([
+            { action: 'press', element: el },
+            { action: 'wait', ms: 1200 },
+            'release',
+          ]);
+
+          await exitModal('Cancel');
+        });
+        it('should long press on an element with duration through longPress-wait-release', async function () {
+          const el = await driver.$(`~${BTN_OK_CNCL}`);
+          await driver.touchAction([
+            { action: 'longPress', element: el },
+            { action: 'wait', ms: 1200 },
+            'release',
+          ]);
+
+          await exitModal('Cancel');
+        });
+        it('should long press on arbitrary coordinates', async function () {
+          const el = await driver.$(`~${BTN_OK_CNCL}`);
+          const loc = await el.getLocation();
+          const size = await el.getSize();
+
+          await driver.touchAction([
+            {
+              action: 'longPress',
+              x: loc.x + size.width / 2,
+              y: loc.y + size.height / 2,
+            },
+            { action: 'wait', ms: 500 },
+            'release',
+          ]);
+
+          await exitModal('OK');
+        });
+      });
+    });
+    // TODO: Need a scrollable screen.
```

Comment:
I marked skip since this fails at line 172. Looks like this test was created for the device with a lower resolution. But now all the items in the top menu are shown. Is it better to add a new screen for scroll testing sake to UIKitCatalog?
<img width="383" alt="スクリーンショット 2022-12-28 16 25 11" src="https://user-images.githubusercontent.com/8503062/209774872-b2d753ed-711a-4d0c-ae8b-1512832c7204.png">

Reply:
for now it would be enough to just add a comment on why this test is disabled. Also, I believe some screen should be still scrollable in one of these submenus

Reply:
I see! Then let me keep it as it is for now. Actually, I cannot find a scrollable screen in those submenus. Please let me know if you found something good. Thank you.